CHANGELOG.md

```diff
@@ -12,10 +12,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 -   [#613](https://github.com/SciLifeLab/Sarek/pull/613) - Add Issue Templates (bug report and feature request)
 -   [#614](https://github.com/SciLifeLab/Sarek/pull/614) - Add PR Template
 -   [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Add presentation
--   [#615](https://github.com/SciLifeLab/Sarek/pull/615) - Update documentation
+-   [#616](https://github.com/SciLifeLab/Sarek/pull/616) - Update documentation
 -   [#620](https://github.com/SciLifeLab/Sarek/pull/620) - Add `tmp/` to `.gitignore`
 -   [#625](https://github.com/SciLifeLab/Sarek/pull/625) - Add [`pathfindr`](https://github.com/NBISweden/pathfindr) as a submodule
--   [#639](https://github.com/SciLifeLab/Sarek/pull/639) - Add a complete example analysis to docs
+-   [#629](https://github.com/SciLifeLab/Sarek/pull/629) - Add a complete example analysis to docs
```

Comment:
Actually this one really was the #639

Reply:
OK, we should keep the PR names in the  description lines, even if they are long. Now I am confused since both #629 and #639 are docs-related PRs, I guess it is #639 here since that is "Added" not only updated, is it right?

Reply:
It has not to be the PR name.
It could be a shorter description, since a PR can fall into `Added`, Changed`, `Removed` or even `Fixed` category at the same time.
For me `Added` is anything new
and `Changed` is anything updated